templates/include/glean/glean.1.schema.json

```diff
@@ -73,8 +73,8 @@
       },
       "required": [
         "range",
-	"bucket_count",
```

Comment:
nit: should we align `bucket_count` and `values` to `range`?

Reply:
Weird.  It looked aligned in my editor.  I'll do a tab-to-space conversion and see if that helps.